### PR TITLE
Namadillo: Creating Docker setup

### LINKED ISSUE #926: Namadillo: Create Dockerfile
We should create a Dockerfile to run Namadillo production build

### PR DESCRIPTION
**Pull request by pedrorezende**
This PR introduces the first version of a Dockerfile for Namadillo

Closes #926

### CODE REVIEW

File: apps/namadillo/package.json [hunk 1]
```diff
@@ -22,6 +22,7 @@
     "framer-motion": "^11.0.28",
     "idb-keyval": "^6.2.1",
     "invariant": "^2.2.4",
+    "io-ts": "^2.2.21",
```
Comment (jurevans) [inline apps/namadillo/package.json:25]: @pedrorezende Seems that all of the CI is failing due to this. Maybe we just need to update the `yarn.lock` file?

Comment (pedrorezende) [reply apps/namadillo/package.json:25]: Thanks @jurevans I missed the yarn.lock on my commit!

**Review by Fraccaman — APPROVED** [review]

**Comment by Fraccaman** [comment]
@pedrorezende thanks, lgtm!

**Review by emccorson — APPROVED** [review]
Looks good! The only question I have is about how this will be used. If we get bug reports from the GitHub releases built through the CI, it should be easy to track down which version is the problem, but I'm not sure whether that will be the case for Docker builds from main. But I don't know, maybe it's not worth worrying about.

**Comment by pedrorezende** [comment]
I think the idea is to have a different option to host Namadillo, once Docker is widely used. Also, the CI is generating the build, so we can do whatever we want with that package. On the other hand, the Docker image configures everything.

**Pull request merged**